docs/docs/development/new-shield.md

```diff
@@ -92,10 +92,10 @@ config ZMK_KEYBOARD_NAME
 endif
 ```
 
-Similarly to defining the halves of a split board in `Kconfig.shield` it is important to set the `ZMK_KEYBOARD_NAME` for each half of a split keyboard.
-You'll also want to set which half is the central side. Most boards set it to the left.
-Then on the peripheral half, you'll want to turn USB on so that it shows USB status on displays properly.
-Finally, you'll want to turn on the split option for both sides. This can all be seen below.
+For split keyboards, `Kconfig.defconfig` needs to specify a few more options.
+First, it configures one side as the central, which is usually the left side.
```

Comment:
Pedantic: the code sample below sets the name and central side in the opposite order as is described here.

Reply:
Good point. I reworded it, please have a look again.

Reply:
Looks reasonable to me.